tests/testing/concerns/test-makes-http-requests.php

```diff
@@ -104,29 +120,14 @@ public function test_rest_api_route_error() {
 			->assertStatus( 404 );
 	}
 
-	public function test_cookie_test() {
-		$_SERVER['__cookie_test'] = false;
-
-		$this->app['router']->get(
-			'/set-cookie',
-			function() {
-				$_SERVER['__cookie_test'] = request()->cookie( 'test-cookie', 'fallback-value' );
-
-				return 123;
+	public function test_multiple_requests() {
+		// Re-run all test methods on this class in a single pass.
```

Comment:
This is an interesting test... 🤔

Reply:
Yeah, I ran into some issues where the tests could be run in isolation but not one after another. Figured I might as well re-run them all in a single go!